src/python/director/transformUtils.py

```diff
@@ -157,28 +153,18 @@ def transformFromPose(position, quaternion):
     '''
     Returns a vtkTransform
     '''
-    rotationMatrix = np.zeros((3,3))
-    vtk.vtkMath.QuaternionToMatrix3x3(quaternion, rotationMatrix)
 
-    mat = np.eye(4)
-    mat[:3,:3] = rotationMatrix
+    mat = transformations.quaternion_matrix(quaternion)
```

Comment:
This replaces VTK's QuaternionToMatrix3x3 with the equivalent conversion routine provided by transformations.py